bin/mm-format-ns-glucose.sh

```diff
@@ -13,19 +13,22 @@ Format Medtronic glucose data into something acceptable to Nightscout.
 EOT
 
 NSONLY=""
-test "$1" = "--oref0" && NSONLY="this.glucose = this.sgv" && shift
+test "$1" = "--oref0" && NSONLY="| .glucose = .sgv" && shift
```

Comment:
what does this `| .` syntax do?

Reply:
The `|` character is similar to a Unix `|`.  The output of the filter to its left is the input to the filter to its right.

The `.` character refers to the current filter input object.

The snippet above doesn't provide the context because NSONLY is simply tagged onto the end of a long set of filters if its presence is required. The pipe allows it to take the output of the rest of the filters.